content/en/blog/community/spinkube-kind-rd/index.md

```diff
@@ -0,0 +1,328 @@
+---
+date: 2024-04-01
+title: "SpinKube on KinD: Rancher Desktop super powers"
+linkTitle: SpinKube on KinD
+description: >
+  Run SpinKube on KinD thanks to Rancher Desktop WASM implementation.
+author: Nuno "the Corsair" Do carmo ([@nunixtech](https://twitter.com/nunixtech))
+resources:
+  - src: "**.{png,jpg}"
+    title: "Image #:counter"
+---
+
+The goal of this guide is show a way to bring [SpinKube](https://www.spinkube.dev/) to [KinD](https://kind.sigs.k8s.io/) without the need of a custom image, like the [SpinKube on k3d](https://www.spinkube.dev/docs/spin-operator/quickstart/) example.
+
+Instead, the Rancher Desktop Spin plugin will be used alongside KinD cluster configuration.
+
+Finally, this guide will have 3 main sections covering the following:
```

Comment:
I would re-phrase this one to

"This guide consists of three major sections, in which we will cover the following:"

Reply:
I kept "three major sections covering the following:".
As I wrote using active voice (at least tried to 😅 ), the `we` is avoided.

Reply:
Addressed in [bc85128](https://github.com/spinkube/documentation/pull/143/commits/bc851287ff5d8cc87a364a71989b2e79515dcb79)